lib/s3-html/log.html

```diff
@@ -165,7 +165,7 @@
                   </button>
                 </div>
                 <div class="pf-v6-c-toggle-group__item">
-                    <button class="pf-v6-c-toggle-group__button pf-m-selected" type="button" data-filter="c-filter-only-failed-retried">
+                    <button class="pf-v6-c-toggle-group__button" type="button" data-filter="c-filter-only-failed-retried">
```

Comment:
I'm not sure what this does, but does that mean that you can't "select" the "Show all" and friends buttons? I find the current behaviour very confusing: You can select and unselect them, but unselecting does nothing. Also, "only show failed' is a bit useless -- one often needs to compare the retries, and there shouldn't be so many failures. Can the whole thing please just collapse to a Switch that says "Show only failures"?

Reply:
Ack will do